static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -326,7 +326,9 @@ define([
                         resolve();
                     }
 
-                    if (abVideoAutoplay.variant === 'autoplay') {
+                    var contentType = config.page.contentType;
+                    var isArticleOrLiveBlog = contentType === 'Article' || contentType === 'LiveBlog';
+                    if (abVideoAutoplay.variant === 'autoplay' && isArticleOrLiveBlog) {
```

Comment:
actually that's not enough, you have to use isVariant('abVideoAutoplay', 'autoplay') to make sure you do all right the checks